app/client/components/consent/PrivacySettings.tsx

```diff
@@ -82,17 +87,75 @@ const buttonStyles = css`
   margin: 0 6px;
 `;
 
+const integStyles = css`
+  margin-right: 5px;
+  border: none;
+  border-radius: 15px;
+  padding: 5px 10px;
+  background-color: ${palette.neutral[6]};
+`;
+
+const cleanGuPurposeList: GuPurposeList = {
```

Comment:
Any reason we're naming this `cleanGuPurposeList` and not just `guPurposeList`? It got me looking for a (non-existant) `dirtyGuPurposeList` and the code to clean it.

Reply:
I see why you've done this as `guPurposeList` is used elsewhere, once `cleanGuPurposeList` has been parsed.